week3/src/views/GamePage.jsx

```diff
@@ -0,0 +1,194 @@
+import { React, useState, useEffect, useRef } from "react";
+import styled, { css } from "styled-components";
+import 버논 from "../img/버논.jpg";
+import 서강준 from "../img/서강준.jpg";
+import 원빈 from "../img/원빈.jpg";
+import 차은우 from "../img/차은우.jpg";
+import 남주혁 from "../img/남주혁.jpg";
+import 로운 from "../img/로운.png";
+import 이수혁 from "../img/이수혁.jpg";
+import 유승호 from "../img/유승호.jpg";
+import 대결 from "../img/vs.png";
+import ResultPage from "../components/Result.jsx";
+
+let gameInfo = [
```

Comment:
```suggestion
const gameInfo = [
```

배열은 참조형 객체이기 때문에, `const` 로 선언해주어도 프로퍼티를 변경할 수 있어!
안정성을 위해 const로 선언해주는 것도 좋겠따!
[참고 아티클](https://morohaji.tistory.com/55?category=940314)

Reply:
헐랭 진짜?ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ몰랐어 고마워!!